app/assets/javascripts/map/views/LayersCountryView.js

```diff
@@ -0,0 +1,158 @@
+/**
+ * The layers filter module.
+ *
+ * @return singleton instance of layers fitler class (extends Backbone.View).
+ */
+define([
+  'backbone',
+  'underscore',
+  'chosen',
+  'map/presenters/LayersCountryPresenter',
+  'map/collections/CountryCollection',
+  'handlebars',
+  'text!map/templates/layersCountry.handlebars',
+], function(Backbone, _, chosen, Presenter, CountryCollection, Handlebars, tpl) {
+
+  'use strict';
+
+  var LayersCountryModel = Backbone.Model.extend({});
+
+
+  var LayersCountryView = Backbone.View.extend({
+
+    el: '#country-layers',
+
+    template: Handlebars.compile(tpl),
+
+    model: new (Backbone.Model.extend({
+      country: null,
+      countryName: null,
+      countryLayers: null
+    })),
+
+    events: {
+      'change #layers-country-select' : 'changeIso',
+      'click #layers-country-reset' : 'resetIso'
+    },
+
+    initialize: function(map, countries) {
+      // Init presenter
+      this.presenter = new Presenter(this);        
+      this.map = map;
+      this.countries = countries;
+      
+      this.render();
+      this.listeners();
+    },
+
+    render: function() {
+      this.$el.html(this.template({
+        countries: this.countries.toJSON(),
+        country: this.model.get('country'),
+        countryName: this.model.get('countryName') || 'Country',
+        countryLayers: this.model.get('countryLayers')
+      }));
+      
+      this.cache();
```

Comment:
I don't agree... I want to standarize all the common view functions along the app ("cache", "render", "listeners"). Maybe you are right with the "fixLegibility". For example, you use "render" function and you don't name it "renderView".

For example "cache": we always cache selectors, but this function also cache variables that aren't selectors.